new_buildpack.go

```diff
@@ -119,7 +88,50 @@ func createBinScript(path, name, contents string, c *Client) error {
 			return err
 		}
 
-		c.logger.Infof("    %s  bin/%s", style.Symbol("create"), name)
+		if c != nil {
+			c.logger.Infof("    %s  bin/%s", style.Symbol("create"), name)
```

Comment:
When would the client be `nil`? Did you find this in your testing?

Reply:
It's `nil` when creating the temporary inline buildpack because we don't want to emit the log line. But when this is used for `pack buildpack new` it passes a Client.